app/locales/he.js

```diff
@@ -26,59 +26,59 @@ export default {
     }
   },
   project: {
-    loading: 'Loading project',
+    loading: 'פרויקט בטעינה',
     disclaimer: 'This project has been built using the Zooniverse Project Builder but is not yet an official Zooniverse project. Queries and issues relating to this project directed at the Zooniverse Team may not receive any response.',
-    fieldGuide: 'Field Guide',
+    fieldGuide: 'מדריך מעשי',
     about: {
-      header: 'About',
+      header: 'אודות',
       nav: {
-        research: 'Research',
-        results: 'Results',
-        faq: 'FAQ',
-        education: 'Education',
-        team: 'The Team',
+        research: 'מחקר',
+        results: 'תוצאות',
+        faq: 'שאלות נפוצות',
+        education: 'מידע חינוכי',
+        team: 'צוות',
       }
     },
     nav: {
-      about: 'About',
+      about: 'אודות',
       adminPage: 'Admin page',
-      classify: 'Classify',
-      collections: 'Collect',
+      classify: 'קיטלוג',
+      collections: 'אוספים',
       exploreProject: 'Explore Project',
       lab: 'Lab',
-      recents: 'Recents',
-      talk: 'Talk',
+      recents: 'לאחרונה',
+      talk: 'שיחה',
       underReview: 'Under Review',
       zooniverseApproved: 'Zooniverse Approved'
     },
     classifyPage: {
-      dark: 'dark',
-      light: 'light',
-      title: 'Classify',
-      themeToggle: 'Switch to %(theme)s theme'
+      dark: 'כהה',
+      light: 'בהיר',
+      title: 'קיטלוג',
+      themeToggle: 'החלף לרקע %(theme)s'
     },
     home: {
       organization: 'Organization',
       researcher: 'Words from the researcher',
-      about: 'About %(title)s',
+      about: 'על הפרויקט %(title)s',
       metadata: {
-        statistics: '%(title)s Statistics',
-        classifications: 'Classifications',
-        volunteers: 'Volunteers',
-        completedSubjects: 'Completed Subjects',
-        subjects: 'Subjects'
+        statistics: 'סטטיסטיקות',
+        classifications: 'קיטלוגים',
+        volunteers: 'משתתפים',
+        completedSubjects: 'פריטים הושלמו',
+        subjects: 'פריטים'
       },
       talk: {
-        zero: 'No one is talking about **%(title)s** right now.',
-        one: '**1** person is talking about **%(title)s** right now.',
-        other: '**%(count)s** people are talking about **%(title)s** right now.'
+        zero: 'אף אחד לא משוחח על הפרויקט כרגע',
+        one: 'אדם 1 משוחח על הפרויקט כרגע',
+        other: '**%(count)s** אנשים משוחחים על הפרויקט כרגע '
```

Comment:
Usually we insert the project title here with `**%(title)s**`. See the [Spanish](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/locales/es.js#L71) and [French](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/locales/fr.js#L73) translations for example. Is there a reason the "title" interpolation was not used here? Does it not make sense for the Hebrew syntax?

Reply:
thanks @wgranger. There are issues with syntax when interpolating with a RTL language.  I could not succeed in achieving the proper order of syntax when interpolating 2 elements (i.e. how many people are participating in the talk + project title), so I left 1 interpolation (how many people are participating in the talk) and left the title generic translated to "this project".

Reply:
Yes, I think there's an issue with how it would/wouldn't flip the incoming rtl project title string consistent with this translated string.

Reply:
That makes sense. I'm not very familiar with Hebrew, so I appreciate the clarification.